eng/pipelines/coreclr/pr.yml

```diff
@@ -171,7 +171,7 @@ jobs:
     jobTemplate: /eng/pipelines/coreclr/templates/crossgen-comparison-job.yml
     buildConfig: checked
     platforms:
-    - Linux_arm
+    # - Linux_arm Return this when https://github.com/dotnet/runtime/issues/1282 is fixed.
```

Comment:
I would suggest increasing the timeout, instead of disabling the submission entirely.

Reply:
Done, thanks